docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3.7'
+services:
+  media:
+    build:
+      context: .
+      target: app
+    ports:
+      - 4001
```

Comment:
This is dynamic port exposure - meaning it will be different port every time - do you suggest hardcoding this port? I prefer this way especially because of the conflicts in ports you might have. Still this requires usage of `docker-compose port` / `docker port` commands.

Reply:
If you just do a docker-compose up then it will be port `4001` to access the media server. So even if we work with dynamic ports if we work with some standard ports like the ones i mentioned above i think it will make development easier. And if somebody pulls this image they know it's port 5000.